components/ShadowElevations/src/MDCShadowElevations.h

```diff
@@ -14,70 +14,91 @@
  limitations under the License.
  */
 
+#import <Availability.h>
 #import <CoreGraphics/CoreGraphics.h>
+#import <Foundation/Foundation.h>
+
+// This macro is introduced in Xcode 9.
+#ifndef CF_TYPED_ENUM // What follows is backwards compat for Xcode 8 and below.
+#if __has_attribute(swift_wrapper)
+#define CF_TYPED_ENUM __attribute__((swift_wrapper(enum)))
+#else
+#define CF_TYPED_ENUM
+#endif
+#endif
+
+/**
+ */
+NS_SWIFT_NAME(ShadowElevation)
+typedef CGFloat MDCShadowElevation CF_TYPED_ENUM;
```

Comment:
Ah, I see what you were referring to about const now. These do still need to be const values.

```objc
typedef const CGFloat MDCShadowElevation CF_TYPED_ENUM;
```

Reply:
Done.